libcxx/include/__new/new_handler.h

```diff
@@ -15,14 +15,10 @@
 #  pragma GCC system_header
 #endif
 
-#if defined(_LIBCPP_ABI_VCRUNTIME)
-#  include <new.h>
-#else
```

Comment:
Can we split this into a separate patch? This doesn't seem to be directly related to modules.

Reply:
it is, because `<new>` does not define `get_new_handler()` and that stops standard modules from building, the change was requested by @frederick-vs-ja to simplify the previous solution.

Reply:
Anyway, can we create another PR for `get_new_handler` and then proceed in this PR?

Reply:
> Anyway, can we create another PR for `get_new_handler` and then proceed in this PR?

Sure, I'll open a new PR that has the changes of the last two commits that fix `get_new_handler` and reference this PR.